packages/blockchain-api/src/config.ts

```diff
@@ -41,7 +38,8 @@ export const DEPLOY_ENV = (process.env.DEPLOY_ENV as string).toLowerCase()
 export const EXCHANGE_RATES_API = (process.env.EXCHANGE_RATES_API as string).toLowerCase()
 export const { EXCHANGE_RATES_API_ACCESS_KEY } = getSecrets(DEPLOY_ENV)
 export const BLOCKSCOUT_API = (process.env.BLOCKSCOUT_API as string).toLowerCase()
-export const FIREBASE_DB = process.env.FIREBASE_DB
+export const FIREBASE_PROJECT_ID = process.env.FIREBASE_PROJECT_ID
```

Comment:
I wonder if there's a way for App Engine to give us this value so we don't need to pass it in via a config? Have you tried logging process.env to see what's already included?
Def not a blocker

Reply:
Yes I looked at this and it's available as `GOOGLE_CLOUD_PROJECT` but then thought, it's better to be explicit for when we test locally.
Thoughts?